### ISSUE TITLE
Line numbers don't match code lines

### ISSUE THREAD

**Issue by lgarcin** [hobarrera/sphinx-autorun#10]
The lines in the output of `.. runblock:: pycon`  are too small. When use with the `:linenos:` flag, the line numbers don't match code lines.

For example
```
.. runblock:: pycon
    :linenos:

    >>> a = (1, 'a', 2, 'b', 3, 'c', 4, 'd', 5, 'e', 6, 'f')
    >>> a[3]
    >>> a[2:]
    >>> a[:5]
    >>> a[-3]
    >>> a[2:9:2]
    >>> a[8:1:-3]
```
produces
![image](https://user-images.githubusercontent.com/1644777/30246601-70739012-95ff-11e7-9a0b-dcc61c766e8b.png)

**Comment by hobarrera** [comment]
I don't think this can trivially fixed: this sphinx extension just generates the block, and the line numbers are later added by sphinx. We've no control over line numbers, so there's not much we can do on our end.
